src/ticks/syms.c

```diff
@@ -65,7 +65,7 @@ static int symbol_compare(const void *p1, const void *p2)
     return s2->address - s1->address;
 }
 
-static char read_symbol_buf[2048];
+static char read_symbol_buf[8192];
```

Comment:
One particularly long class simply didn't fit
```
F/Users/desertkun/Documents/Work/zx-sandbox/client/include/client_map.h$map_client_object_t[({0}S:S$object$0_0$0({8}STmap_object_t:S),Z,0,0)({8}S:S$block_bottom_m_2$0_0$0({1}SC:U),Z,0,0)({9}S:S$block_bottom_m_1$0_0$0({1}SC:U),Z,0,0)({10}S:S$block_bottom_p_0$0_0$0({1}SC:U),Z,0,0)({11}S:S$block_bottom_p_1$0_0$0({1}SC:U),Z,0,0)({12}S:S$block_bottom_p_2$0_0$0({1}SC:U),Z,0,0)({13}S:S$block_bottom_p_3$0_0$0({1}SC:U),Z,0,0)({14}S:S$block_right_1$0_0$0({1}SC:U),Z,0,0)({15}S:S$block_right0$0_0$0({1}SC:U),Z,0,0)({16}S:S$block_right1$0_0$0({1}SC:U),Z,0,0)({17}S:S$block_left_1$0_0$0({1}SC:U),Z,0,0)({18}S:S$block_left0$0_0$0({1}SC:U),Z,0,0)({19}S:S$block_left1$0_0$0({1}SC:U),Z,0,0)({20}S:S$target_x$0_0$0({2}SI:U),Z,0,0)({22}S:S$target_y$0_0$0({2}SI:U),Z,0,0)({24}S:S$jump_potential$0_0$0({1}SC:U),Z,0,0)({25}S:S$force$0_0$0({1}SC:U),Z,0,0)({26}S:S$sprite_data$0_0$0({2}DG,SC:U),Z,0,0)({28}S:S$sprites$0_0$0({2}DG,STobject_sprites_t:S),Z,0,0)({30}S:S$flags$0_0$0({1}SC:U),Z,0,0)({31}S:S$dirty$0_0$0({1}SC:U),Z,0,0)({32}S:S$offset_x$0_0$0({1}SC:U),Z,0,0)({33}S:S$offset_y$0_0$0({1}SC:U),Z,0,0)]
```
That's over 1024 and you have to essentially double that because of encoding.